package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Next.js SDK for signing in with Auth0",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "browser": "dist/instance.browser.js",
```

Comment:
This should actually link to a file that implements the same exports as the `main` file, except for the browser.

Reply:
Updated